Docker/test/curl/test_curl.sh

```diff
@@ -26,6 +33,4 @@ if [ -z "$SKIP_WDQS" ]; then
 
     check_if_up "$WDQS_FRONTEND_SERVER" "/"
 
-fi
-
-
+fi
```

Comment:
Could we agree on having a `\n` at the end of every file? Does vscode support that? Otherwise I think we will have lots of those whitespace changes back and forth in the future.

POSIX wants that too https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206

Reply:
I would prefer a required linebreak at the end of every file. I don't have my settings in VS Code on this project do that and was actually removing them when I found them as it seemed the default in this project was to not have a trailing \n. Doesn't seem to be any linting about it, and like I say, I prefer (is more familiar to me) to have a final \n. Probably won't change anything here now though as we sort this out more generally.

Reply:
follow up in https://phabricator.wikimedia.org/T343802